app/src/main/java/com/mapbox/vision/examples/activity/main/MainActivity.kt

```diff
@@ -48,7 +48,9 @@ import com.mapbox.vision.performance.ModelPerformanceRate
 import com.mapbox.vision.safety.VisionSafetyManager
 import com.mapbox.vision.safety.core.VisionSafetyListener
 import com.mapbox.vision.safety.core.models.CollisionDangerLevel
-import com.mapbox.vision.safety.core.models.CollisionDangerLevel.*
```

Comment:
ktlint

Reply:
\+

Reply:
Well, turns out `ktlint` isn't applied in Teaser yet. I've submitted an issue to apply it https://app.zenhub.com/workspaces/vision-sdk-5ca52e6c027e3e7234ac7512/issues/mapbox/mapbox-vision/1700